### ISSUE TITLE
Firefox / Edge / IE playback?

### ISSUE THREAD

**Issue by TV5Dev** [streamroot/videojs5-hlsjs-source-handler#11]
Hi,

I'm playing around using this player, and it's working great on Chrome / Safari.
I am however experiencing playback issues on Edge / Firefox.
Following URL playback works is very stable on Chrome / Safari:
https://asia.tv5monde.com/Test/streamroot/SampleI.html

On Edge 25.10586.0.0, it seems to loop and keep downloading the lower quality playlist (not downloading any segments)

On Windows Firefox 46.0.1 playback works for a while, until I receive the error: "The media playback was aborted due to a corruption problem or because the media used features your browser did not support." and also seems to loop and keep downloading the lower quality playlist (not downloading any segments)

Could it be that there is something wrong with the playlist format / segments?

Thanks for your help.

**Comment by TV5Dev** [comment]
On IE11, behavior seems to be the same as on Edge after adding polyfills from issue #12

**Comment by AxelDelmas** [comment]
It seems to be a hls.js issue. Did you test your stream in a vanilla hls.js ? Here's the hls.js demo page: http://dailymotion.github.io/hls.js/demo/
If you reproduce your issues in this page, then you should file a bug directly in hls.js repo: https://github.com/dailymotion/hls.js/issues

**Comment by TV5Dev** [comment]
Hi Axel,

I have made a small change so only primary encoder streams are in the playlist file at following location, but the issue persists: https://asia.tv5monde.com/Test/streamroot/SampleI2.html

I did test the livestream on the hls.js page with default settings and playback is working.
permalink: http://dailymotion.github.io/hls.js/demo/?src=https%3A%2F%2Ftv5mondehlslive-i.akamaihd.net%2Fhls%2Flive%2F250600%2F4792245510001-5%2Ftv5plusinfo%2Findex.m3u8&enableStreaming=true&autoRecoverError=true&enableWorker=true&levelCapping=-1&defaultAudioCodec=undefined

I am using Firefox 47.0 on Windows 10.

Thanks again.

**Comment by AxelDelmas** [comment]
Ok I took a look and actually the same error happens in hls.js demo page, but it has a error recovery mechanism actually implemented in #13 .
I gave #13 a try with your stream, and I run in another issue: apparently, videojs detects error directly on the media element, and will display the message "The media playback was aborted due to a corruption problem or because the media used features your browser did not support." 

@jhilden I think we need to disable default error handling behaviour of videojs for on top of implementing an error recovery mechanism. I found this issue that seems related: https://github.com/brightcove/videojs-errors/issues/21

I tried to add the following in the source handler in order to clear the error in videojs, but it doesn't seem to have any effect:

``` javascript
tech.player_.on('error', function() {
    tech.player_.error(null);
});
```
